conbench/entities/commit.py

```diff
@@ -228,14 +228,9 @@ def backfill_default_branch_commits(
     for commit_info in commits_to_try:
         commit_info["github"]["branch"] = default_branch
         commit_info["github"]["fork_point_sha"] = commit_info["sha"]
-        try:
-            commit = Commit.create_github_context(**commit_info)
+        commit = Commit.create_github_context(**commit_info)
```

Comment:
About the removal of the exception handler:

While the row conflict would here now be eliminated as a reason for this statement to raise an exception -- the individual database transaction can of course still fail with a whole range of transient errors. Without exception handler in the loop body that means that an individual transient error may prematurely crash the `for commit_info in commits_to_try` loop.

A good way to address this is to issue 1 instead of N database transactions, as suggested by my higher level review comment on this pull request! :)

Reply:
I'm not too worried about prematurely crashing this loop, since the backfill will be attempted again later, but I agree that one transaction lessens the chance of something going wrong and also probably speeds up this function a little. I'll change it now!

Reply:
Okay, I changed to a single transaction. Let me know what you think!

Reply:
Thanks!